lib/http/headers.rb

```diff
@@ -94,7 +94,18 @@ def to_a
     #
     # @return [String]
     def inspect
-      "#<#{self.class} #{to_h.inspect}>"
+      "#<#{self.class} #{pretty_print}>"
+    end
+
+    # Returns a printable representation of headers.
+    #
+    # @param [String] separator
+    # @return [String]
+    def pretty_print(separator = ", ")
+      to_h.map do |name, value|
+        value = value.join("; ") if value.respond_to?(:join)
+        "#{name}: #{value}"
+      end.join(separator)
```

Comment:
Hum, OK.
I used `", "` separator to `inspect` method to return in a single line. Is it OK `inspect` methods return multiple lines?
Just confirming.

Same approach with `inspect` method in `HTTP::Request` and `HTTP::Response` classes?